contrib/build-container.sh

```
@@ -92,6 +95,14 @@ get_options() {
                 missing_requirement $1
             fi
             ;;
+        --cuda-version)
+            if [ "$2" ]; then
+                CUDA_VERSION=$2
+                shift
+            else
+                missing_requirement $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)